src/olympia/devhub/tests/test_views_subscription.py

```diff
@@ -321,6 +321,8 @@ def test_get_redirects_to_main_page(self, retrieve_mock):
 
     @mock.patch("olympia.devhub.views.retrieve_stripe_checkout_session")
     def test_get_records_payment_once(self, retrieve_mock):
+        self.subscription.promoted_addon.approve_for_version(
```

Comment:
Why are the changes in this file needed for this patch?

Reply:
Those tests were triggering a resign once that was added in an earlier patch - it just wasn't apparent until now.  I could have created the task user for the tests to pass, but they didn't need to do the resign for the functionality being tested, so I pre-approved the version to skip the resigning instead (mocking the resigning task would have worked too).